src/main/java/domain/Cars.java

```diff
@@ -0,0 +1,40 @@
+package domain;
+
+import java.util.List;
+
+public class Cars {
+
+    private final List<Car> carList;
+    private final MoveStrategy moveStrategy;
+
+    public Cars(List<Car> carList, MoveStrategy moveStrategy) {
+        this.carList = carList;
+        this.moveStrategy = moveStrategy;
+    }
+
+    public static Cars of(List<String> carNames, MoveStrategy moveStrategy) {
+        List<Car> carList = carNames.stream()
+                .map(Car::new)
+                .toList();
+        return new Cars(carList, moveStrategy);
+    }
+
+    public void move() {
+        carList.forEach((car) -> {
+            car.move(moveStrategy.isMove());
+        });
```

Comment:
[의견] `car`에 불필요한 괄호를 없앨 수 있을 것 같아요.

```suggestion
        carList.forEach(car -> car.move(moveStrategy.isMove()));
```

Reply:
처음에는 `{}`안에 있는 구문이 여러 줄이었다가, 차례대로 줄여나갔습니다.
마지막 한 줄까지 줄이고 나서 더 간단하게 나타낼 수 있다는 점을 놓쳤네요;;

주신 의견에 맞춰 수정하도록 하겠습니다~